### DISCUSSION TITLE
Feed in Compensation Calculation

### DISCUSSION THREAD

**Discussion by DerRonnyUK** [nathanmarlor/foxess_modbus#662] [Q&A] [answered]
Apologies for the stupid sounding question, but where can I specify the amount I get paid for the feed in? Maybe I am blind?

**Comment by jamesog** [comment, marked as answer]
Go to Settings > Dashboards > Energy

On the Electricity Grid section, configure Return to grid with the Feed-in Total sensor, and either set a static amount or use an entity (e.g. from https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/) that has the tariff amount.

<img width="389" alt="image" src="https://github.com/user-attachments/assets/f6522ed1-d27c-4ad1-bff8-1e9f1b2b161d">

<img width="427" alt="image" src="https://github.com/user-attachments/assets/8d0bce3f-bbc1-4407-9083-0e45c5614ed8">

**Reply by DerRonnyUK** [reply]
thanks James - it was to obvious for me to find ... thanks again
